### PR TITLE
Increase the read/write timeout of tests in large value scenario

### LINKED ISSUE #999: Flaky test TestReplicationWithLimitSpeed
```
--- FAIL: TestReplicationWithLimitSpeed (10.10s)
    client.go:72: 
        	Error Trace:	/Users/runner/work/incubator-kvrocks/incubator-kvrocks/tests/gocase/integration/replication/client.go:72
        	            				/Users/runner/work/incubator-kvrocks/incubator-kvrocks/tests/gocase/integration/replication/replication_test.go:209
        	Error:      	Received unexpected error:
        	            	i/o timeout
        	Test:       	TestReplicationWithLimitSpeed
```

Unstable test. @git-hulk perhaps release the read timeout for the specific case.

_Originally posted by @tisonkun in https://github.com/apache/incubator-kvrocks/issues/998#issuecomment-1279942166_

Check run: https://github.com/apache/incubator-kvrocks/actions/runs/3258497049/jobs/5350638195

### PR DESCRIPTION
**Pull request by git-hulk**
Resolves #999

### CODE REVIEW

File: tests/gocase/integration/replication/replication_test.go [hunk 1]
```diff
@@ -204,7 +204,10 @@ func TestReplicationWithLimitSpeed(t *testing.T) {
 		"rocksdb.target_file_size_base": "1",
 	})
 	defer master.Close()
-	masterClient := master.NewClient()
+	masterClient := master.NewClientWithOption(&redis.Options{
+		ReadTimeout:  10 * time.Second,
+		WriteTimeout: 10 * time.Second,
```
Comment (tisonkun) [inline tests/gocase/integration/replication/replication_test.go]:
```suggestion
```

`WriteTimeout` is default to `ReadTimeout`.

**Review by tisonkun — DISMISSED** [review]
LGTM. One minor comment.

**Comment by tisonkun** [comment]
Please reformat code :)

**Comment by git-hulk** [comment]
> Please reformat code :)

Done, I saw the lint error

**Review by tisonkun — APPROVED** [review]
I just notice that it's the 1000th issue/PR, lol.

**Comment by git-hulk** [comment]
> I just notice that it's the 1000th issue/PR, lol.

😆

**Pull request merged**